hazelcast/src/main/java/com/hazelcast/internal/metrics/impl/ProbeBuilderImpl.java

```diff
@@ -104,4 +117,11 @@ public String metricName() {
             method.register(this, source);
         }
     }
+
+    private String prefixedMetricName(String metricName) {
```

Comment:
I would simplify this with a ternary operator.

Reply:
Addressed. Thanks for noticing this.